drivers/wdm/audio/legacy/wdmaud/entry.c

```diff
@@ -31,24 +31,17 @@ WdmAudInitWorkerRoutine(
     /* get device extension */
     DeviceExtension = (PWDMAUD_DEVICE_EXTENSION)DeviceObject->DeviceExtension;
 
-
-    if (DeviceExtension->FileObject == NULL)
+    /* find available sysaudio devices */
+    Status = WdmAudOpenSysAudioDevices(DeviceObject, DeviceExtension);
+    if (!NT_SUCCESS(Status))
     {
-        /* find available sysaudio devices */
-        Status = WdmAudOpenSysAudioDevices(DeviceObject, DeviceExtension);
-        if (!NT_SUCCESS(Status))
-        {
-            DPRINT1("WdmAudOpenSysAudioDevices failed with %x\n", Status);
-            return;
-        }
+        DPRINT1("WdmAudOpenSysAudioDevices failed with %x\n", Status);
+        return;
     }
 
-
```

Comment:
readd this newline. no need for cosmetic changes here.

Reply:
Fixed (I think).

Reply:
no it's not fixed, now that line contains unintended spaces that were not present before.

Reply:
How do I fix it?

Reply:
What lines do I delete?